include/common/core/arch_config.hpp

```diff
@@ -115,20 +115,22 @@ struct register_attr_t {};
 
 template <grf_mode grf_num_mode, gpu_arch arch_tag>
 struct client_register_attr_base_t {
+  static constexpr uint32_t reg_in_bytes = 32;
```

Comment:
iGPU is different?

Reply:
git rebase error~ igpu is 32 and other is 64